server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -566,11 +567,56 @@ public final class IndexSettings {
 
     public static final Setting<String> INDEX_MERGE_ON_FLUSH_POLICY = Setting.simpleString(
         "index.merge_on_flush.policy",
-        MERGE_ON_FLUSH_DEFAULT_POLICY,
+        DEFAULT_POLICY,
         Property.IndexScope,
         Property.Dynamic
     );
 
+    public static final Setting<String> INDEX_MERGE_POLICY = Setting.simpleString("index.merge.policy", DEFAULT_POLICY, policy -> {
+        if (!(policy.isEmpty()
```

Comment:
The empty policy should not be accepted I believe

Reply:
have removed it